evap/student/templates/student_vote.html

```diff
@@ -64,8 +64,9 @@ <h3 class="mb-3">
                            data-set-spinner-icon="span-set-evaluation-language-{{ language_code }}"
                            title="{{ language_name }}"
                            role="button"
-                           href="{% url 'student:vote' evaluation.pk %}?language={{ language_code }}"
-                           class="btn btn-sm btn-light{% if evaluation_language == language_code %} active{% endif %}">
+                           href="{{ request.path }}?language={{ language_code }}"
+                           class="btn btn-sm btn-light{% if evaluation_language == language_code %} active{% endif %}"
+                           id="chg-lang">
```

Comment:
The indentation is fixed now, but it looks like you accidentally added a new `id="chg-lang">` attribute that looks like it's not needed to me?


```suggestion
                           class="btn btn-sm btn-light{% if evaluation_language == language_code %} active{% endif %}">
```